packages/content/src/attributes/types/strings/AbstractPhoneNumber.ts

```diff
@@ -4,14 +4,16 @@ import { AbstractString } from "../AbstractString";
 
 export abstract class AbstractPhoneNumber extends AbstractString {
     @serialize()
-    @validate({ min: 3, max: 100, regExp: new RegExp(/^[\d+\-x#*()/[\] ]{3,100}$/) })
+    @validate({ min: 3, max: 100, regExp: new RegExp(/^[\d+\-() ]{3,100}$/) })
     public override value: string;
 
     public static override get valueHints(): ValueHints {
         return super.valueHints.copyWith({
             min: 3,
             max: 100,
-            pattern: "/^[d+-x#*()/[] ]{3,100}$/"
+            // prettier-ignore
+            // eslint-disable-next-line no-useless-escape
```

Comment:
it should not be necessary to have these two ignores.

Reply:
prettier is erasing my `\` and eslint complains about useless-escapes. Not sure how I could do it differently

Reply:
wtb double backslash?

Reply:
would break the regex in dart... I could do triple backslash 😄  but that would cause the same problem